RecoMET/METProducers/src/PFMETProducer.cc

```diff
@@ -94,8 +94,9 @@ namespace cms {
     event.getByToken(rhoToken_, rho);
 
     //Compute the covariance matrix and fill it
+    double sumPt;
```

Comment:
Doesn't this need initialization = 0?

Reply:
You're right, thanks for the catch! It's initialized with 0 in the other producers, but somehow missing there. I'll fix it.